update.json

```diff
@@ -0,0 +1,4 @@
+{
+  "html_url": "https://github.com/CanonicalLtd/multipass/releases/tag/v0.9.0",
```

Comment:
Why do you need both `json` and `yaml`? Surely the multipass tool only needs to consume one of them. I'm not opinionated about which format you use, but both seems unnecessary.

Reply:
(Although, if you're interested in my opinion, I would suggest `json` is the better format if it's intended purely for the `multipass` tool to consume)

Reply:
The json is only meant to help us transition. If all goes well, it will be complete very soon and the json can be removed.